### ISSUE TITLE
Bitmex not accepting decimals for quantity...

### ISSUE THREAD

**Issue by hexdump2002** [ccxt/ccxt#2559]
- OS: Windows
- Programming Language version: Python
- CCXT version:  1.12.154
- Exchange: Bitmex
- Method: Just try to place an order with an amount with decimals. Example 1.323

Is there any possibility to have this trait in the "has" object of the exchange description? This way we will be able to convert to int if the exchange does not allow decimals or whatever we need.

Cheers.

**Comment by hexdump2002** [comment]
Hi again,

The link you posted is really nice and I think that it is the solution to my problems. The only problem is that bitmex exchange has neither precision nor limits definitions. 

Can we do something with it?

THanks

**Comment by kroitor** [comment]
@hexdump2002 you're right, my apologies. BitMEX does not have precision and limit info filled yet, but I'll add it shortly, and will let you know. Thx!
